docs/guide/onboarding-library.md

```diff
@@ -1,9 +1,158 @@
 # Onboarding Library
+As an Ethereum enabled site developer, sending users offsite to install MetaMask presents challenges. Most notably, you must inform the user to return to your site and refresh their browser after the installation. Your site will detect the user's newly installed MetaMask extension only after that refresh. We at MetaMask care deeply about user experience, and we knew that this workflow needed to be improved.
 
-MetaMask now allows sites to register as onboarding the user, so that the user is redirected back to the initiating site after onboarding. This is accomplished through the use of the [metamask-onboarding library](https://github.com/MetaMask/metamask-onboarding).
+MetaMask now provides a [metamask-onboarding library](https://github.com/MetaMask/metamask-onboarding) designed to improve and simplify the onboarding experience. The new library exposes an API to initiate the onboarding process. In the process, it registers your site as the origin of the onboarding request. MetaMask will check for this origin after the user completes the onboarding flow. If it finds an origin, the final confirmation button of the MetaMask onboarding flow will indicate that the user will be redirected back to your site. The wording of this button helps prevent phishing attempts, by highlighting that a redirect is taking place to an untrusted third party.
 
-At the end of onboarding, the text of the button will show the origin of the site the user is about to be redirected to. This is intended to help prevent phishing attempts, as it highlights that a redirect is taking place to an untrusted third party.
+## Getting Started
+1. install @metamask/onboarding using npm or yarn.
+2. import the Onboarding Library or include it in your page.
 
+```javascript
+// As an ES6 module
+import MetaMaskOnboarding from '@metamask/onboarding';
+// Or as an ES5 module
+const MetamaskOnboarding = require('@metamask/onboarding');
+```
+
+If you'd prefer you can instead include the prebuilt ES5 bundle that ships with the library:
+```html
+  <script type="text/javascript" src="./metamask-onboarding.bundle.js"></script>
+```
+
+3. create a new instance of the Onboarding library
+
+```javascript
+const onboarding = new MetaMaskOnboarding();
+```
+
+4. kick off the onboarding process.
+
+```javascript
+onboarding.startOnboarding();
+```
+
+## Examples
+### Basic Usage
+```javascript
+const onboarding = new MetamaskOnboarding();
+onboarding.startOnboarding();
+```
+
+### Using React
+```jsx
+import MetaMaskOnboarding from '@metamask/onboarding';
+import * as React from 'react';
```

Comment:
Nit: This seems a little unconventional? 🤷‍♂️ Since `react` does have a default export.

```suggestion
import React from 'react';
```

Reply:
Bad habit from following dated typescript tutorials when I was learning! Without certain tsconfig flags it used to be required to import this way.